packages/base/src/asset-registries/Illustrations.js

```diff
@@ -13,18 +14,58 @@ const registerIllustration = (name, { dialogSvg, sceneSvg, spotSvg, set, title,
 	});
 };
 
+const registerIllustrationLoader = async (illustrationName, loader) => {
+	loaders.set(illustrationName, loader);
+};
+
+const _loadIllustrationCollectionOnce = async illustrationName => {
+	let temp;
+	if (illustrationName.startsWith("Tnt")) {
+		illustrationName = illustrationName.substring(3);
+		temp = `Tnt${illustrationName}`;
+	} else {
+		temp = illustrationName;
+	}
+	if (!loaders.has(illustrationName.replace("Tnt", ""))) {
+		throw new Error(`No loader registered for the ${illustrationName} illustration. You probably forgot to import the "AllIllustrations.js".`);
+	}
+	const loadIllustrations = loaders.get(illustrationName);
+	return loadIllustrations(temp);
```

Comment:
The name of the method suggests it's going to fetch an illustration once, and that's how it should be.
But, it's not how it currently works. You need to store the returned promises by `loadIllustrations` per illustration and reuse it, when the illustration is fetched twice or more. 

See `const iconCollectionPromises = getSharedResource("SVGIcons.promises", new Map());` in  assets-registries/Icons.js

Reply:
You are right, I fixed it.